parser/src/main/java/com/plhal/ares/parser/ParserRepositoryImpl.java

```diff
@@ -52,23 +56,26 @@ public Firma najdiFirmu(String ico) {
 
         } catch (ParserConfigurationException e) {
 
-            e.printStackTrace();
+            log.error("***** ParserConfigurationException occured: ", e);
```

Comment:
Tady warn, protoze ERROR bych bral spise vec, ktera zpusobi pad aplikace. Tady vracite null a aplikace se tvari, ze je vse v poradku. Je to proste ve stylu, ze je dobre o tom vedet, ale soucasne se pocita, ze se to stane a ignoruje. Zadna oprava nebude nasledovat - stejne si to uzivatel pripadne provola znova.